src/apps/main/preload.js

```diff
@@ -368,5 +373,36 @@ contextBridge.exposeInMainWorld('electron', {
 
     return () => ipcRenderer.removeListener(eventName, callbackWrapper);
   },
+  antivirus: {
+    isAvailable: async () => {
+      return await ipcRenderer.invoke('antivirus:is-available');
```

Comment:
If you are using `async` and `await` but returning directly, unless you want to try catch something inside the function - which is not the case -, it's unnecessary, either use async-await and return in the next line or remove it completely. 

In any case, you can still type the function as `Promise<whatever>`